packages/terra-alert/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+Changelog
+=========
+
+0.1.0 - (July 5, 2017)
+------------------
```

Comment:
Please switch `0.1.0 - (July 5, 2017)` to `Unreleased`
When we release v1 for this component, we'll add the date. We are planning to release components on a weekly cycle on Tuesdays.